components/header/mainNav/__snapshots__/index.test.jsx.snap

```diff
@@ -1,5 +1,22 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`MainNav renders Group and Privacy for people with the feature flag turned on 1`] = `
+<DocumentFragment>
+  <div
+    class="PodBrowser-main-nav-container"
+  >
+    <nav
+      class="PodBrowser-main-nav"
+      data-testid="main-nav"
+    >
+      <ul
```

Comment:
This is empty, it should have the Privacy and Group tabs along with all the others that aren't feature flagged.